src/main/js/util.js

```diff
@@ -69,3 +68,9 @@ export const writeSettings = (doc, output) => {
 export const trim = (elem) => {
   return typeof elem === 'string' ? elem.trim() : elem
 }
+
+export const trimCsvFile = (fileContent) =>
```

Comment:
* normalizeCsv
* `.trim().split(/("[^"]+")/g).map((v,i) => i % 2 ? v : v.replace(/\s*,\s*/g, ',')).join('')` — this snippet properly removes __all__ redundant spaces